src/main/java/uk/gov/hmcts/reform/pdf/service/config/DeprecatedApiConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.pdf.service.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.handler.MappedInterceptor;
+import uk.gov.hmcts.reform.api.deprecated.DeprecatedApiInterceptor;
+
+@Configuration
+public class DeprecatedApiConfiguration {
+
+    @Bean
+    public MappedInterceptor deprecatedApi() {
```

Comment:
could the library to do this for us instead?

Reply:
It is certainly doable. Could be a feature request for the project.

Reply:
worth updating readme for https://github.com/hmcts/reform-api-standards if this is required atm.